src/screens/Followings/FollowersScreen.js

```diff
@@ -90,7 +90,7 @@ function FollowersScreen() {
   }, []);
   return (
     <View style={{flex: 1}}>
-      {isAndroid ? <StatusBar translucent={false} /> : null}
+      {isAndroid ? <StatusBar translucent={false} barStyle={'light-content'} /> : null}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made to the `StatusBar` component is good for improving the visibility of the status bar content on Android devices. However, it's important to ensure that the `'light-content'` style doesn't conflict with the rest of your app's UI design. If there are screens where this style isn't appropriate, you might need to dynamically adjust the `barStyle` based on the current screen or theme.
```diff
- {isAndroid ? <StatusBar translucent={false} /> : null}
+ {isAndroid ? <StatusBar translucent={false} barStyle={'light-content'} /> : null}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->